### PR TITLE
TA4 -- Basic scoring implemented

### PR DESCRIPTION
**Pull request by drawnfutures**

### CODE REVIEW

File: BlockBlock/Assets/Scripts/GameManager.cs [hunk 1]
```diff
@@ -356,4 +362,21 @@ private void DisableBlocks()
             block.GetComponent<BoxCollider2D>().enabled = false;
         }
     }
+
+    // TODO: move to own ScoreManager if scoring gets more complex
+    // Updates the score based on number of rows/cols cleared
+    private void UpdateScore(int amount)
+    {
+        score += amount * 10;
```
Comment (VinTangible) [inline BlockBlock/Assets/Scripts/GameManager.cs]: Maybe make the 10 multiplier a constant at the top?

File: BlockBlock/Assets/Scripts/Score.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    // Start is called before the first frame update
+    void Start()
+    {
+        scoreText.text = "";
+    }
+
+    // Update is called once per frame
+    private void Update()
+    {
+        scoreText.text = "" + GameManager.instance.getScore().ToString();
```
Comment (VinTangible) [inline BlockBlock/Assets/Scripts/Score.cs]: What is the empty string for?

Comment (drawnfutures) [reply BlockBlock/Assets/Scripts/Score.cs]: Leftover from previous implementation; changing.

**Comment by VinTangible** [comment]
Oh btw, we are changing the line endings from CRLF to LF so there are merge conflicts. I'd suggest change your local files to LF first and then pull master (and if there are merge conflicts still then accept your local changes instead of incoming changes)

**Pull request merged**